README.md

```diff
@@ -67,6 +67,7 @@ $ purs-tidy generate-config --arrow-first --unicode-never --operators .tidyopera
 ## Editor Support
 
 * [Vim](#vim)
+* [Spacemacs](#spacemacs)
```

Comment:
I think this should move to the top to be alphabetical and easier to find as a result

Reply:
Yup! Done